configure.py

```diff
@@ -1481,8 +1487,9 @@ def _generate_target_sources_list(fd, target_name, target):
         fd.write('set(%s\n' % target_name)
         sorted_sources = sorted(target['sources'].keys())
         for source in sorted_sources:
-            fd.write('    ${CMAKE_CURRENT_LIST_DIR}%s%s\n' % (os.sep, os.path.normpath(source)))
+            fd.write(('    ${CMAKE_CURRENT_LIST_DIR}%s%s\n' % (os.sep, os.path.normpath(source))).replace('\\', '\\\\'))
```

Comment:
Why keep the os.sep and backward slashes in this place? Maybe we should have a cmake_normpath function in Python, that does os.path.normpath and replace("\\", "/").

This makes it easier to review the generated cmake file and to_cmake_paths becomes obsolete, right?